control/control.yaml

```diff
@@ -74,18 +74,17 @@ components:
           enum: [start, stop]
 
     Flows.Update:
-      description: A container of flows with associated properties to be updated without affecting the transmit state
+      description: A container of flows with associated property to be updated without affecting the transmit state
       type: object
-      required: [property_names]
+      required: [property_name, flows]
       properties:
-        property_names:
-          description: Flow properties to be updated without affecting the transmit state 
+        property_name:
```

Comment:
shouldn't property_names be an array of enums ?

Reply:
updated (and retained name as _property_names_)